packages/create/src/cli/add/annotations-to-odata.ts

```diff
@@ -73,7 +73,8 @@ async function addAnnotationsToOdata(basePath: string, simulate: boolean, yamlPa
                     filePath: answers.filePath,
                     namespaces,
                     serviceUrl: dataSources[answers.id].uri
-                }
+                },
+                isCommand: true
```

Comment:
Method is reused via control property editor to create annotation change to differentiate cli call and cpe call added this variable